### PR TITLE
Configured Firebase with React Native for Android

### LINKED ISSUE #10: Connection with database
## REQUIREMENTS / DESCRIPTION

> As a React Native developer, I want to establish a connection with Firebase as the database for the project so that I can leverage Firebase's features for data storage, retrieval, and synchronization.

```[tasklist]
### Tasks
- [x] Set up Firebase project
- [x] Install and Initialize Firebase SDK
- [x] Establish connection with Firebase
```

### PR DESCRIPTION
**Pull request by Eduardoespl**
# Description

This pull request includes the necessary configuration to integrate Firebase in a React Native project targeting Android. Here are the changes made:
Set up Firebase by creating a Firebase project and obtaining the required configuration details such as API keys and project ID.
Installed the Firebase SDK and its dependencies using npm or yarn.
Updated the Android project configuration files (android/app/build.gradle, android/app/src/main/java/.../MainApplication.java) to include the necessary Firebase initialization code.

### CODE REVIEW

**Comment by Tole15** [comment]
LGTM

File: AppVuelos/android/app/google-services.json
Comment (BrandonVG) [inline AppVuelos/android/app/google-services.json:1]: You should hide this file or the file's content

**Review by BrandonVG — APPROVED** [review]
LGTM

**Review by juperameza — DISMISSED** [review]
Lgtm

**Review by Tole15 — APPROVED** [review]

**Review by Tole15 — APPROVED** [review]

File: AppVuelos/android/app/build.gradle
Comment (juperameza) [inline AppVuelos/android/app/build.gradle:1]: This file should be in your git ignore

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]

File: AppVuelos/android/app/build.gradle
Comment (juperameza) [reply AppVuelos/android/app/build.gradle:1]: NVM just watch out not to include sensitive information

**Pull request merged**